members.yaml

```diff
@@ -226,3 +226,10 @@
   LastName: Su
   affiliation: Tusimple
   email: lei.su@tusimple.com
+
+- github: jzp1025 
```

Comment:
Can you sort this file by github id please? it looks like jinxingwang and suleisl2000 are also out of place.